src/domain/event/modal/UnenrolModal.tsx

```diff
@@ -74,8 +74,8 @@ const UnenrolModal = ({
       // eslint-disable-next-line no-console
       console.error(error);
       // TODO: KK-280 Handle errors nicely
-      toast(t('registration.submitMutation.errorMessage'), {
-        type: toast.TYPE.ERROR,
+      toast.error(t('registration.submitMutation.errorMessage'), {
+        type: error,
       });
```

Comment:
```suggestion
      toast.error(t('registration.submitMutation.errorMessage'));
```
In other files toast was used this way (without options object).